tasks/sep/base-005-fp-holocene-upgrade/.env

```diff
@@ -0,0 +1,3 @@
+ETH_RPC_URL="https://ethereum-sepolia.publicnode.com"
+OWNER_SAFE=0x0fe884546476dDd290eC46318785046ef68a0BA9
```

Comment:
I see that Base Sepolia still uses a non-nested Safe. @mds1 is that something you'd also like to migrate to a nested setup to mirror the Base setup on Mainnet?

Reply:
That kind of change would need to be discussed with Henri and Anika